### PR TITLE
Add .gitignore and pom.xml

### PR DESCRIPTION
**Pull request by 51-code**
.gitignore copied from PTH-10

pom.xml includes these plugins:

- enforcer
- formatter (spotless)
- rat
- maven-surefire-plugin

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/target/
+/.settings/
+/.vscode/
+/.idea/
+/*.iml
+/.classpath
+/.factorypath
+/.flattened-pom.xml
+fuzzer/**
+output.log
```
Comment (kortemik) [inline .gitignore:10]: remove fuzzer and output.log, not used here

File: pom.xml
Comment (kortemik) [inline pom.xml:1]: as this is a new project, please add checkstyle and jacoco too

**Review by kortemik — CHANGES_REQUESTED** [review]
workflow files are missing as well

File: .gitignore (hunk 1 above)
Comment (51-code) [reply .gitignore]: Removed.

File: pom.xml
Comment (51-code) [reply pom.xml:1]: Added the plugins.

**Review by kortemik — APPROVED** [review]

**Pull request merged**